plonky3/src/circuit_builder.rs

```diff
@@ -257,13 +215,27 @@ impl<'a, T: FieldElement> PowdrCircuit<'a, T> {
     }
 }
 
+/// An extension of [Air] allowing access to the number of fixed columns
```

Comment:
I think we already have this in BaseAir in our fork

Reply:
I think it's called `preprocessed_width`

Reply:
Oops sorry this was leftover from the old fork